src/rtp/id.rs

```diff
@@ -238,6 +238,33 @@ impl SeqNo {
     pub fn roc(&self) -> u64 {
         self.0 >> 16
     }
+
+    /// Generate a SeqNo from a u32, relative this SeqNo.
+    ///
+    /// This can be used to extend an extended sequence number from a RR (32-bit)
```

Comment:
You mean a _partial_ sequence number?

Reply:
no.. I mean an extended seqno.... so RTP uses 16-bit in the headers, but 32-bit extended SeqNo in the RR... and Str0m has this 64bit extended extended seq no